contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -289,7 +278,10 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         for (uint256 i; i < length; ++i) {
             TeleporterMessageReceipt memory receipt = teleporterMessage.receipts[i];
             _markReceipt(
```

Comment:
might be gas savings but not necessary, we could pass in the receipt directly to `_markReceipt`, instead of individual fields then we don't need the extra variable declaration above

Reply:
Nice clean up catch 🙏